static/js/click_handlers.js

```diff
@@ -556,18 +557,28 @@ export function initialize() {
         });
 
     function do_render_buddy_list_tooltip(elem, title_data) {
-        elem.tooltip({
-            template: render_buddy_list_tooltip(),
-            title: render_buddy_list_tooltip_content(title_data),
-            html: true,
-            trigger: "hover",
-            placement: "bottom",
-            animation: false,
-        });
-        elem.tooltip("show");
+        let placement = "left";
+        if (window.innerWidth < Number(media_breakpoints.md_min.slice(0, -2))) {
```

Comment:
Can we do something with a function ro hide the "slice of `px` and parse to integer" complexity?

Maybe just `static/js/css_variables.js` should export two objects, `media_breakpoints_px` and `media_breakpoints_int` or something?